bower.json

```diff
@@ -4,15 +4,18 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-	"angular": "*",
-	"jquery": "*",
-	"bootstrap": "*",
-	"angular-route": "*",
-	"angular-resource": "*",
-	"angular-animate": "*",
-	"hammerjs": "*",
-	"underscore": "*",
-	"socket.io-client": "*",
-   	"angular-sanitize": "*"
+    "angular": "*",
```

Comment:
Why do you need the last library? I've never heard of it before. Please inform me regarding it.